airbyte-server/src/main/java/io/airbyte/server/handlers/SourceDefinitionsHandler.java

```diff
@@ -285,16 +289,6 @@ public void deleteSourceDefinition(final SourceDefinitionIdRequestBody sourceDef
     configRepository.writeStandardSourceDefinition(persistedSourceDefinition);
   }
 
-  public void deleteCustomSourceDefinition(final SourceDefinitionIdWithWorkspaceId sourceDefinitionIdWithWorkspaceId)
```

Comment:
is this change supposed to be part of this PR? Or did it sneak in from your other branch for consolidating API endpoints?

Reply:
ah true